src/cleanup.ts

```diff
@@ -0,0 +1,14 @@
+import * as core from '@actions/core'
+import Docker from 'dockerode'
+
+export async function run(): Promise<void> {
+  try {
+    const docker = new Docker()
+    await docker.pruneNetworks({filters: {until: '24h'}})
```

Comment:
Just double-checking the thinking on 24h, our fastest schedule is 'daily', so assuming only using Version Updates this is more or less 'clean up the last run' which seems 👍🏻 

Is there any reason for doing this _more often_ than 24 hours?